### PR TITLE
fix(issues): Clean up various components in issue details

### PR DESCRIPTION
**Pull request by dcramer**
- Dont show expand action on Suspect Commits with only 1 value
- Correct alignment on expand action on Suspect Commits
- Completely remove clientIP from SDK interface
- Move outdated SDK warning to footer of issue details

### CODE REVIEW

**Comment by dcramer** [comment]
@mitsuhiko i removed all of the old client_ip code rather than relying just on stripping of the data

File: src/sentry/api/serializers/models/event.py [hunk 1]
```diff
@@ -58,11 +58,6 @@ def get_attrs(self, item_list, user, is_public=False):
             sdk_interface = item.interfaces.get('sdk')
             if sdk_interface:
                 sdk_data = sdk_interface.get_api_context()
-                # we restrict SDK information to superusers due to the nature of
-                # privacy laws and IP sensitivity
-                # TODO(dcramer): make this respect 'enhanced privacy' org flag
-                if not user.is_superuser:
```
Comment (dcramer) [inline src/sentry/api/serializers/models/event.py]: this is removed from the get_api_context() call

**Comment by dcramer** [comment]
Before:

screen shot 2018-06-13 at 3 00 31 pm

After:

screen shot 2018-06-13 at 3 00 40 pm

File: src/sentry/event_manager.py [hunk 2]
```diff
@@ -502,9 +502,6 @@ def to_values(v):
         elif client_ip and (is_public or data.get('platform') in add_ip_platforms):
             data.setdefault('sentry.interfaces.User', {}).setdefault('ip_address', client_ip)
 
-        if client_ip and data.get('sdk'):
```
Comment (dcramer) [inline src/sentry/event_manager.py]: this stops storing the data at all for future events

File: src/sentry/interfaces/sdk.py [hunk 3]
```diff
@@ -57,7 +57,6 @@ def to_python(cls, data):
         kwargs = {
             'name': trim(name, 128),
             'version': trim(version, 128),
-            'client_ip': data.get('client_ip'),
```
Comment (dcramer) [inline src/sentry/interfaces/sdk.py]: this prevents the interface from maintaining the data on save

File: src/sentry/interfaces/sdk.py [hunk 4]
```diff
@@ -82,7 +81,6 @@ def get_api_context(self):
         return {
             'name': self.name,
             'version': self.version,
-            'clientIP': self.client_ip,
```
Comment (dcramer) [inline src/sentry/interfaces/sdk.py]: never return it in api context

**Review by billyvg — APPROVED** [review]
To clarify, we're removing ip reporting from our sdks?

**Comment by dcramer** [comment]
@billyvg client_ip was generally something the server captured on its own (since it came from the SDK itself). We already are stripping it to avoid the GDPR mess, so this just removes some useless code.

**Pull request merged**